### DISCUSSION TITLE
Add Markdown Kroki suport

### DISCUSSION THREAD

**Discussion by leandro-costa** [quarto-dev/quarto-cli#6900] [Feature Requests]
Kroki provides a unified API with support for BlockDiag (BlockDiag, SeqDiag, ActDiag, NwDiag, PacketDiag, RackDiag), BPMN, Bytefield, C4 (with PlantUML), D2, DBML, Ditaa, Erd, Excalidraw, GraphViz, Mermaid, Nomnoml, Pikchr, PlantUML, Structurizr, SvgBob, Symbolator, UMLet, Vega, Vega-Lite, WaveDrom, WireViz.

https://kroki.io/examples.html#uml-diag

**Comment by cderv** [comment]
Thanks for suggestion. As others feature requests, we will consider it for future development, according to our priorities for each future versions. 

In the meantime, I believe an extension could be made for this. (Mkdoc has an [plugin](https://github.com/AVATEAM-IT-SYSTEMHAUS/mkdocs-kroki-plugin)  for it for example. [Sphinx](https://github.com/sphinx-contrib/kroki) too)

For references, 
* node.js tool for this: https://github.com/yuzutech/kroki
* Julia package: https://github.com/bauglir/Kroki.jl
* Jupyter Magic: https://pypi.org/project/jupyter-kroki-magic/

**Reply by crisbour** [reply]
I made a [Bytefield extension for Quarto](https://github.com/crisbour/bytefield) as I did not find an alternative extensions that supports it

**Reply by cderv** [reply]
Cool ! Thanks for sharing !

**Comment by mcanouil** [comment]
For reference, as listed on https://m.canouil.dev/quarto-extensions/:

- https://github.com/resepemb/quarto-kroki
- https://github.com/fermarsan/quarto-kroki
